Formula/go.rb

```diff
@@ -5,8 +5,8 @@ class Go < Formula
 
   stable do
     if Hardware::CPU.arm?
-      url "https://golang.org/dl/go1.16beta1.src.tar.gz"
-      sha256 "48e032c8cf71af4dc8119a29ee829c4fbd5265e32fd012564d4a70bb207695c1"
+      url "https://golang.org/dl/go1.16rc1.src.tar.gz"
+      sha256 "6a33569f9d0d21db31614086cc2a4f0fbc683b41c1c53fb512a1341ce5763ff5"
       version "1.15.7"
```

Comment:
You probably need to add `revision 1` to the end of this `if` block, or else anyone who has `go` installed won't get the updated version unless they do `brew reinstall go`.

Check this with `brew audit --strict go` first, though; I might be mistaken about where the `revision` line needs to go here.

Reply:
If I add the revision I get the error "Error: No available formula or cask with the name" 🤔

Reply:
Oof. Ok. Try moving the `revision 1` line just outside the `stable` block. Or just basically put it somewhere just before the install `block`. `brew audit --strict --fix go` should hopefully help you put it where it belongs.

Reply:
okay, that seems to work. Thank you!